servers/republik/modules/crowdfundings/graphql/resolvers/User.js

```diff
@@ -123,6 +123,25 @@ module.exports = {
       }
 
       memberships = await resolveMemberships({ memberships, pgdb })
+
+      // Checks if there is a pending pledge on a users active membership.
+      const hasPendingPledges = await pgdb.public.query(`
+        SELECT "pledges".* FROM "pledges"
+
+        JOIN "pledgeOptions"
+          ON "pledges"."id" = "pledgeOptions"."pledgeId"
+          AND "pledgeOptions"."membershipId" = :activeMembershipId
+
+        WHERE "pledges"."status" = 'WAITING_FOR_PAYMENT'
+        ;
+      `, { activeMembershipId: memberships.find(m => m.active).id || 0 })
```

Comment:
Would it not make more sense to move that down to [L161](https://github.com/orbiting/backends/pull/163/files#diff-cee2cfea33e6969e45a1d051cdc6c0e3R161) and use eligableMemberships to find the active one?

Reply:
Would keep it there to return early. Thinking: If there is a pending pledge, finding eligible memberships is useless. In fact, we might even move it up a little further - no need to "resolve memberships".

Reply:
but it has to be a prolong pledge to a extendable membership no? anyway, it should always be the same, so it's ok for me.